cli/src/spend_utils.rs

```diff
@@ -181,12 +215,14 @@ where
                     SpendAmount::AllForAccountCreation {
                         create_account_min_balance,
                     } => create_account_min_balance,
-                    SpendAmount::All | SpendAmount::RentExempt => 0,
+                    SpendAmount::All | SpendAmount::Available | SpendAmount::RentExempt => 0,
                 }
             } else {
                 match amount {
                     SpendAmount::Some(lamports) => lamports,
-                    SpendAmount::AllForAccountCreation { .. } | SpendAmount::All => from_balance,
+                    SpendAmount::AllForAccountCreation { .. }
+                    | SpendAmount::All
+                    | SpendAmount::Available => from_balance,
```

Comment:
nit: It's slightly confusing that `from_balance` doesn't necessarily mean the balance in the `from` account.

We could maybe rename this to `from_balance_available` or `from_balance_transferable`, but I don't feel great about either option.

Reply:
I went with `from_account_transferable_balance`... thoughts?